src/core/nonbonded_interactions/nonbonded_interaction_data.cpp

```diff
@@ -77,28 +77,31 @@
 #include <cstdlib>
 #include <cstring>
 
+#include "electrostatics_magnetostatics/coulomb.hpp"
+#include "electrostatics_magnetostatics/dipole.hpp"
+
 /****************************************
  * variables
  *****************************************/
 int max_seen_particle_type = 0;
 std::vector<IA_parameters> ia_params;
 
-#if defined(ELECTROSTATICS) || defined(DIPOLES)
-Coulomb_parameters coulomb = {
 #ifdef ELECTROSTATICS
+Coulomb_parameters coulomb = {
```

Comment:
Move to `coulomb.hpp`

Reply:
should i also move the `enum`?

Reply:
Yes, everything. Same for dipolar, so that only `#include "coulomb.hpp"` is needed to use it.